lib/archethic_web/controllers/faucet_controller.ex

```diff
@@ -0,0 +1,80 @@
+defmodule ArchEthicWeb.FaucetController do
+  @moduledoc false
+
+  alias ArchEthic.TransactionChain.{
+    Transaction,
+    TransactionData,
+    TransactionData.Ledger,
+    TransactionData.UCOLedger
+  }
+
+  alias ArchEthic.Crypto
+
+  use ArchEthicWeb, :controller
+
+  def index(conn, __params) do
+    conn
+    |> put_resp_header("cache-control", "no-cache, no-store, must-revalidate")
+    |> put_resp_header("pragma", "no-cache")
+    |> put_resp_header("expires", "0")
+    |> render("index.html", address: "")
+  end
+
+  def create_transfer(conn, %{"address" => address}) do
+    case transfer(address, Crypto.default_curve()) do
+      :ok ->
+        conn
+        |> put_flash(:info, "Transferred successfully")
+        |> redirect(to: "/faucet")
+
+      {:error, _} ->
+        conn
+        |> put_flash(:error, "Unable to transfer")
+        |> render("index.html", address: address)
+    end
+  end
+
+  defp transfer(
+         address,
+         curve
+       )
+       when is_bitstring(address) do
+    gen_seed = Application.get_env(:archethic, __MODULE__)[:seed]
```

Comment:
Because it's not subject to change, I think you can use compilation environment using module attributes

Reply:
Using the config as module attribute like this?

```elixir
defmodule ArchEthicWeb.FaucetController do
  @external_resource "config/config.exs"
  @seed Application.get_env(:archethic, __MODULE__)[:seed]

  defp transfer do
    @seed
  end
end
```

Reply:
You can indeed use `@seed` attribute to load from the configuration.

But the no need for the `@external_resource` attribute